sdcm/cluster.py

```diff
@@ -1243,8 +1243,15 @@ def _get_tcpdump_logs(self, tcpdump_id):
             self.log.error('Error running tcpdump on lo, tcp port 10000: %s',
                            str(details))
 
+    def _is_storage_virtualized(self):
+        return self.is_docker()
+
     def fstrim_scylla_disks(self):
-        self.remoter.sudo("fstrim -v /var/lib/scylla")
+        if not self._is_storage_virtualized():
+            self.remoter.sudo("fstrim -v /var/lib/scylla")
+        else:
+            LOGGER.warning(
```

Comment:
Would be really good to have an SCT event like following here:
```
TestFrameworkEvent(                                                                 
    source=self.__class__.__name__,                                                 
    source_method='fstrim_scylla_disks',                                         
    message="fstrim'ming of Scylla disks was skipped",                       
    severity=Severity.WARNING,                                                      
).publish_or_dump()  
```

Reason: Debugging SCT we mostly depend on the events, not log messages.